pkg/detect/ospkg/ospkg.go

```diff
@@ -172,20 +176,20 @@ func replaceIndexes(ac criteriaTypes.FilteredCriteria, indexes []int) criteriaTy
 		replaced.Criterias = append(replaced.Criterias, rca)
 	}
 
-	var cos []criteriaTypes.FilteredCriterion
-	for _, co := range ac.Criterions {
-		if len(co.Accepts) == 0 {
+	var cns []criteriaTypes.FilteredCriterion
+	for _, cn := range ac.Criterions {
+		if len(cn.Accepts) == 0 {
 			continue
 		}
 
-		is := make([]int, 0, len(co.Accepts))
-		for _, a := range co.Accepts {
+		is := make([]int, 0, len(cn.Accepts))
+		for _, a := range cn.Accepts {
 			is = append(is, indexes[a])
 		}
-		co.Accepts = is
-		cos = append(cos, co)
+		cn.Accepts = is
+		cns = append(cns, cn)
 	}
-	replaced.Criterions = cos
+	replaced.Criterions = cns
```

Comment:
You should make the same change here too.
https://github.com/MaineK00n/vuls2/pull/84/files#diff-bf1f3d3fc4dd9c7172fda625d2f706506db8624e551ebb6e0adab81affe5dd34R125-R138

Reply:
done https://github.com/MaineK00n/vuls2/compare/60ef7c3d8f84777e42828085dd0746e008187ee2..296a7fd45df1a2a7c22523a475d4b19cb089b837